specification/_global/search/_types/rescoring.ts

```diff
@@ -72,3 +84,14 @@ export enum ScoreMode {
    */
   total
 }
+
+export class LearningToRank {
+  /**
+   * The unique identifier of the trained model uploaded to Elasticsearch
+   */
+  model_id?: string
```

Comment:
```suggestion
  model_id: string
```

Doesn't seem optional according to the server code.